common/workunit/workunit.cpp

```diff
@@ -6787,7 +6787,7 @@ void CLocalWorkUnit::setNodeState(const char *graphName, WUGraphIDType nodeId, W
 }
 IWUGraphStats *CLocalWorkUnit::updateStats(const char *graphName, StatisticCreatorType creatorType, const char * creator, unsigned subgraph) const
 {
-    throwUnexpected();   // Should only be used for persisted workunits
+    return new CWuGraphStats(LINK(p), creatorType, creator, graphName, subgraph);
```

Comment:
this was part of my -wu change and not directly needed by the timer stats change AFAIK